src/resources/views/taxon/_tree.blade.php

```diff
@@ -0,0 +1,19 @@
+@foreach($taxonomies as $taxon)
```

Comment:
It's either `@foreach($taxonomies as $taxonomy)` or `@foreach($taxons as $taxon)`.